### ISSUE TITLE
How do I install disk.frame?

### ISSUE THREAD

**Issue by xiaodaigh** [r-hub/r-minimal#17]
I tried

```
FROM rhub/r-minimal
RUN installr -d -t "libcurl4-openssl-dev zlib-dev" disk.frame
```

but it doesn't work. curl needs `libcurl4-openssl-dev` but it's not found, and `curl` is needed for disk.frame and so is ` zlib-dev`

**Comment by gaborcsardi** [comment]
curl needs `curl-dev`

**Comment by xiaodaigh** [comment]
This issue is solved but now #20 is the issue
